src/libraries/System.Private.CoreLib/src/System/Double.cs

```diff
@@ -354,33 +354,33 @@ public override int GetHashCode()
 
         public override string ToString()
         {
-            return Number.FormatDouble(m_value, null, NumberFormatInfo.CurrentInfo);
+            return Number.FormatFloat(m_value, null, NumberFormatInfo.CurrentInfo);
```

Comment:
Not going to block on this, but we should probably call this `FormatFloatingPoint` instead and it would be good to fix it in a follow up PR.

`Float` is going to be confused with the C# keyword `float`, which should be called `Single` in the BCL API surface area